Updated What's New In zOS 3.1 CD 1Q24 document

cresyi opened the pull request:
Updates made based on feedback. 
Signed-off-by: Cresyi Padilla cpadill@us.ibm.com

v1gnesh commented:
Any chance you can detail what was changed?
Not in 1Q24, but between https://github.com/IBM/IBM-Z-zOS/pull/271 and this PR.

cresyi commented:
@v1gnesh The content was updated based in this feedback:

- In Title: Speaker Notes: Updated. 
- Original Slide 35: This slide about EzNoSQL moved to corrected category  
- Original Slide 75 - the original: This slide about zWIC is right now in category “Systems Management”. Moved to the same category as RMF and Advanced Data Gatherer
- Original Slide 96 + 99: Fixed duplicated content
